packages/vue-dot/src/patterns/FooterBar/CollapseList/CollapseList.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+	<div class="vd-collapse-list my-4">
```

Comment:
```suggestion
	<div class="vd-collapse-list">
```
Il ne faut pas appliquer de margin au composant